### ISSUE TITLE
A11y: Focus Indicator has insufficient contrast in various QuickView elements

### ISSUE THREAD

**Issue by shopwareBot** [shopware/platform#5245]
The Focus indicator for: 

* Image Gallery pagination
* Variant property radio options

Have insufficient contrast against both the active element color and the background.

The focus indicator for:

* ‘Add to Shopping Cart’ button 

Has insufficient contrast against the background (it’s sufficient against the button color).

{panel:bgColor=#deebff}
This also affects the various product page elements, particularly variants
{panel}

**Comment by shopwareBot** [comment]
[public] Shopware automated response: This issue is linked to the internal issue: **CMS-483**



[created from CMS-483, comment 506158]
